edge-modules/azure-monitor/src/module.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Remove this file...looks like it was part of the previous developer's personal config.

Reply:
Removed.